doc/tutorials/introduction/openvino_install/openvino_install.markdown

```diff
@@ -0,0 +1,547 @@
+OpenCV usage with OpenVINO {#tutorial_openvino_install}
+=====================
+
+@prev_tutorial{tutorial_ios_install}
+@next_tutorial{tutorial_display_image}
+
+|    |    |
+| -: | :- |
+| Original author | Aleksandr Voron |
+| Compatibility | OpenCV == 4.x |
+
+@tableofcontents
+
+# Quick start {#tutorial_openvino_install_introduction}
+
+Since 2022.2 release OpenVINO does not provide pre-built OpenCV. If you are using OpenVINO with OpenCV functionality there are 2 approaches how to get it:
```

Comment:
No sure if we need this information in OpenCV.
Relevant place is OpenVINO location instead.